### ISSUE TITLE
About UR 9, 10, 11, 19 and perhaps 12

### ISSUE THREAD

**Issue by mpakarlsson** [daisy/reading-apps-ux-reqs#14]
There's a lot of URs starting with "Screen reader/assistive technology users should be able to..." where these are applicable for all users. I suggest to review the titles for UR 9, 10, 11, 19 and perhaps 12, to see if we can use a more generic approach to capture all users with title starting with "The user should be able to..."

**Comment by KirstenDeHaan** [comment]
Excellent idea. The URs are adapted in the next version of the requirements document.
